core/templates/dev/head/domain/statistics/ExplorationIssueObjectFactory.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating new frontend instances of Exploration
+ *     Issue domain objects.
+ */
+
+oppia.factory('ExplorationIssueObjectFactory', [function() {
```

Comment:
Do we need `fromBackendDict`?

Reply:
Had a different reasoning to not have this in this class, but on second thought, we do need it. Thanks for pointing out.